source/collections/Vector2D.ooc

```diff
@@ -38,6 +40,58 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
```

Comment:
I know that you have probably discussed this with @simonmika but to me this naming seems very confusing.
Let me explain. `columnCount` sounds like a *number of the columns*, while `columnCapacity` sounds like a *capacity of a single column* or *size of a single column*, which is in fact exactly the same as `rowCount` (*number of rows*)...
I suggest using *count* instead of *capacity* in this class, but this is only my opinion.
@davidhesselbom , @simonmika what do you think about it ?